internal-packages/workflow-designer-ui/src/editor/properties-panel/file-node-properties-panel/web-page-file-panel.tsx

```diff
@@ -25,7 +25,7 @@ export function WebPageFilePanel({ node, config }: FilePanelProps) {
 				{/* Fetched Files List */}
 				{files.length > 0 && (
 					<div className="pb-[16px] flex flex-col gap-[8px]">
-						{files.map((file) => (
+						{files.map((file: FileData) => (
```

Comment:
This type annotation can remove since file is inferenced
```suggestion
						{files.map((file) => (
```